legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/project/config/ProjectCreationConfiguration.java

```diff
@@ -148,100 +89,31 @@ public Pattern getArtifactIdPattern()
         return this.artifactIdPattern;
     }
 
-    public boolean isDisallowedType(ProjectType type)
-    {
-        return this.disallowedTypes.contains(type);
-    }
-
-    public String getDisallowedTypeMessage(ProjectType type)
+    @Deprecated
+    @JsonIgnore
+    public Object getDisallowedTypes()
     {
-        return this.disallowedTypeMessages.get(type);
+        return null;
     }
 
     @JsonCreator
-    public static ProjectCreationConfiguration newConfig(@JsonProperty("defaultProjectStructureVersion") Integer defaultProjectStructureVersion, @JsonProperty("groupIdPattern") Pattern groupIdPattern, @JsonProperty("artifactIdPattern") Pattern artifactIdPattern, @JsonProperty("disallowedTypes") Collection<DisallowedType> disallowedTypes)
+    public static ProjectCreationConfiguration newConfig(@JsonProperty("defaultProjectStructureVersion") Integer defaultProjectStructureVersion, @JsonProperty("groupIdPattern") Pattern groupIdPattern, @JsonProperty("artifactIdPattern") Pattern artifactIdPattern)
```

Comment:
I think we need to keep the "disallowedTypes" property for backward compatibility.

Reply:
As discussed, unfortunately, we would not try to keep this backward-compatible